### ISSUE TITLE
Doesn't Work !!!  chrome(canary/dev) v72 with Oreo

### ISSUE THREAD

**Issue by NiNJAD3vel0per** [googlecodelabs/ar-with-webxr#22]
I have two Android device (one is the latest phone(Samsung), another one is Nexus5x with Oreo)

I have to install older version of chrome because of this example limitation.

Finally, Nothing is possible to work....

I lost too much valuable time.

**Comment by NiNJAD3vel0per** [comment]
The browser said, "installing ar module" many for ARCore right? 
Then. Unsupported Browser!!!! 
I double checked Enabling  chrome://flags ;;

**Comment by vogloblinsky** [comment]
hi this version works for me https://www.apkmirror.com/apk/google-inc/chrome-dev/chrome-dev-72-0-3626-14-release/

**Comment by NiNJAD3vel0per** [comment]
@vogloblinsky 

Got it ! Thanks.. I will test it :)

**Comment by ArnaudHambenne** [comment]
For anyone else who landed here in an attempt of getting the samples to work on the newer versions of Chrome: There is an undocumented XRSessionMode called `legacy-inline-ar` which gives you a session with attribute `environmentBlendMode: 'alpha-blend'` without the need of going immersive.

**Comment by ssydz** [comment]
@vogloblinsky Did  you change the samples' code to make it running successfully on  Chrome Dev 72.0.3636.14? I have tried many versions of Chrome Dev and Chrome Canary, but none could work.

**Comment by vogloblinsky** [comment]
@ssydz not really, just forked this codelab to make it available in french : https://github.com/vogloblinsky/webxr-codelab ; but works fine for me with the version mentionned earlier. What kind of error do you have ?

**Comment by ssydz** [comment]
@vogloblinsky it could not get xr attribute from navigator, so the page show unsupported browsers directly. I will try your project late, thank you.

**Comment by vogloblinsky** [comment]
@ssydz i confirm that this url works fine https://vogloblinsky.github.io/webxr-codelab/work/
for an LG G6 with Android 8.0.0 and Chrome Dev 72.0.3626.14 with flags enabled.

**Comment by ssydz** [comment]
@vogloblinsky I have tried the URL on Chrome Dev 72.0.3626.14 with flags enabled again, but it still doesn't work. And whhen I open `chrome://inspect`, I found the `navigator.xr` was undefined. My device is Pixel 2 XL with Android 9. I guess this Android system doesn't support it.

**Comment by winterx** [comment]
> hi this version works for me https://www.apkmirror.com/apk/google-inc/chrome-dev/chrome-dev-72-0-3626-14-release/

Thank you and it works for my device.
I have a Mi 8 with Android 8.1.0, and ARCore installed. First it failed on Chrome Canary, then the dev version you recommend works fine.

**Comment by tomdowning82** [comment]
> > hi this version works for me https://www.apkmirror.com/apk/google-inc/chrome-dev/chrome-dev-72-0-3626-14-release/
> 
> Thank you and it works for my device.
> I have a Mi 8 with Android 8.1.0, and ARCore installed. First it failed on Chrome Canary, then the dev version you recommend works fine.

This link below for me too (Pixel 3a with Chrome Dev 72):
https://googlecodelabs.github.io/ar-with-webxr/work/

But when I download the sample files below, and run a chrome local webserver, i get unsupported device?
https://github.com/googlecodelabs/ar-with-webxr/

Anyone else?

**Comment by winterx** [comment]
> > > hi this version works for me https://www.apkmirror.com/apk/google-inc/chrome-dev/chrome-dev-72-0-3626-14-release/
> > 
> > 
> > Thank you and it works for my device.
> > I have a Mi 8 with Android 8.1.0, and ARCore installed. First it failed on Chrome Canary, then the dev version you recommend works fine.
> 
> This link below work for me too (Pixel 3a with Chrome Dev 72):
> https://googlecodelabs.github.io/ar-with-webxr/work/
> 
> But when I download the sample files below, and run a chrome local webserver, i get unsupported device?
> https://github.com/googlecodelabs/ar-with-webxr/
> 
> Anyone else?

Me too, the online version works but the local version failed

**Comment by opr** [comment]
Anyone having issues running locally, XR requires a secure context, you need to access it using HTTPS.

**Comment by vogloblinsky** [comment]
@opr works fine for me using [Web Server for Chrome](https://chrome.google.com/webstore/detail/web-server-for-chrome/ofhbbkphhbklhfoeikjpcbhemlocgigb)

**Comment by opr** [comment]
Sorry I should have mentioned, if I'm accessing it over my network (i.e. Not localhost) then it needs a secure context
